Documentation/build_qubes_packages.md

```diff
@@ -0,0 +1,320 @@
+# Qubes OS Packages
+
+[Qubes builder](https://www.qubes-os.org/doc/qubes-builder/) can be used to
+prepare new ISO image but also to build customized packages. To do the latter
+we will need PC with Fedora installed (may be any version as builder prepare
+fc32 chroot itself).
```

Comment:
This paragraph sound more like requirements. Also I wonder about new qubes-build which was announced at Qubes OS Summit, are we using it, and if not then the question is why?

Reply:
AFAIK new builder should not be OS-dependent because it leverage containers

Reply:
dunno which one was announced at summit, but we used one pointed in documentation https://www.qubes-os.org/doc/qubes-builder/ which says `In order to use it, you should use an rpm-based distro, like Fedora`

Reply:
@TomaszAIR I guess there are plans to enable builder for other OSes (docker-based solution IIRC). I tried to test it during Qubes OS summit, but had issues with it: https://github.com/QubesOS/qubes-builderv2

Anyway, if for now we use v1 which is recommended it is ok, but in long run I would appreciate to support also Debian-based environment which many community members use.